pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/player/compose/TvPlayerView.kt

```diff
@@ -60,46 +63,55 @@ fun TvPlayerView(
         drawerState = drawerState,
         modifier = modifier
     ) {
-        PlayerSurface(
-            player = player,
-            modifier = Modifier
-                .fillMaxSize()
-                .handleDPadKeyEvents(onEnter = {
-                    visibilityState.show()
-                })
-                .focusable(true)
-        )
-        AnimatedVisibility(
-            visible = visibilityState.isVisible,
-            enter = expandVertically { it },
-            exit = shrinkVertically { it }
-        ) {
-            Box(
+        val error by player.playerErrorAsState()
+        if (error != null) {
+            CompositionLocalProvider(
+                LocalContentColor provides MaterialTheme.colorScheme.onSurface
+            ) {
+                PlayerError(modifier = Modifier.fillMaxSize(), playerError = error!!, onRetry = player::prepare)
+            }
```

Comment:
Since `PlayerError` sets its texts color directly, changing the `LocalContentColor` here does nothing.
```suggestion
            PlayerError(modifier = Modifier.fillMaxSize(), playerError = error!!, onRetry = player::prepare)
```